.github/workflows/canary-integration-test.yml

```
@@ -69,6 +69,18 @@ jobs:
     - name: check-ownerreferences
       run: tests/scripts/github-action-helper.sh check_ownerreferences
 
+    - name: test osd removal jobs
+      run: |
+        kubectl -n rook-ceph delete deploy/rook-ceph-operator
+        kubectl -n rook-ceph delete deploy/rook-ceph-osd-1 --grace-period=0 --force
+        sed -i 's/${FAILED_OSD_IDS}/1/' cluster/examples/kubernetes/ceph/osd-removal.yaml
```

Can we use the existing osd-purge.yaml instead of creating a new yaml?
```suggestion
        sed -i 's/${FAILED_OSD_IDS}/1/' cluster/examples/kubernetes/ceph/osd-purge.yaml
```

---

Yes I totally missed that we had that example already.